backend/aws/lambda/app.py

```diff
@@ -17,6 +20,7 @@
     service_name='secretsmanager',
     region_name=config['secret']['region']
 )
+new_submission = True
```

Comment:
It seems like a few of these methods like `grab_response` and/or `call_deepcite` could be abstracted into a single class and then `new_submission` could be a property. It could be like an `Operations class` or `ResponseHandler class`. But maybe that also should be done in a separate PR.

Reply:
I can agree. I have some code that puts the DB calls in another class, but this would be good too